src/toplevel.lisp

```diff
@@ -137,6 +137,7 @@
    integer-length integerp interactive-stream-p intern
    internal-time-units-per-second intersection invalid-method-error
    invoke-debugger invoke-restart invoke-restart-interactively isqrt
+   js-object js-object-p js-object-signature
```

Comment:
Let's remove this please. I want the "CL" package to contain only standard functionality.

My preference for this is to expose more packages for extensions:

`jscl/ffi` - JS related package

But for now we can use it from the jscl package.

I'll have a look into the packages.

Reply:
Ok